assets/react/front/course/_spotlight-quiz.js

```diff
@@ -10,7 +31,11 @@ window.jQuery(document).ready($=>{
 
         // Evaluate result
         var feedBackMode = $question_wrap.attr('data-quiz-feedback-mode');
```

Comment:
I think we should remove 
var feedBackMode = $question_wrap.attr('data-quiz-feedback-mode');
in the favour of is_reveal_mode()

Reply:
done